File: CS/15_Document/RearrangePageOrder.cs
using Spire.Pdf;
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;

namespace RearrangePageOrder
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            //Create a pdf document
            PdfDocument doc = new PdfDocument();

            //Load from file
            doc.LoadFromFile(@"..\..\..\..\..\..\Data\SampleB_3.pdf");

            //Rearrange the page order
            doc.Pages.ReArrange(new int[] { 1, 0});

            String result="RearrangePageOrder-result.pdf";
            //Save to file
            doc.SaveToFile(result, FileFormat.PDF);

            //Launch the Pdf file
            PDFDocumentViewer(result);

        }
        private void PDFDocumentViewer(string fileName)
        {
            try
            {
                System.Diagnostics.Process.Start(fileName);
            }
            catch { }                    
   
        }
    }
}

File: VB/15_Document/RearrangePageOrder.vb
Imports Spire.Pdf

Namespace RearrangePageOrder
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub button1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles button1.Click
			' Create a new PdfDocument object
			Dim doc As New PdfDocument()

			' Load the PDF document from the specified file path
			doc.LoadFromFile("..\..\..\..\..\..\Data\SampleB_3.pdf")

			' Re-arrange the order of pages in the document by specifying the new page order
			doc.Pages.ReArrange(New Integer() {1, 0})

			' Specify the result file path for saving the modified document
			Dim result As String = "RearrangePageOrder-result.pdf"

			' Save the modified document to the result file in PDF format
			doc.SaveToFile(result, FileFormat.PDF)

			' Close the PDF document
			doc.Close()

			' Launch the Pdf file
			PDFDocumentViewer(result)
		End Sub
		Private Sub PDFDocumentViewer(ByVal fileName As String)
			Try
				Process.Start(fileName)
			Catch
			End Try

		End Sub
	End Class
End Namespace
